From c0e825e5c48b0d783e8314ddf6eb5afa254b6cc8 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Wed, 24 Oct 2012 12:45:16 +0800
Subject: ENGR00230920-2 HDMI AUDIO: export gloable variable in platform
 driver

Export gloable variable in HDMI AUDIO platform driver to support HDMI
AUDIO codec driver's loadable build.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 sound/soc/imx/imx-hdmi-dma.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/imx/imx-hdmi-dma.c b/sound/soc/imx/imx-hdmi-dma.c
index 342f9c8..dc71d52 100644
--- a/sound/soc/imx/imx-hdmi-dma.c
+++ b/sound/soc/imx/imx-hdmi-dma.c
@@ -119,6 +119,7 @@ void hdmi_dma_copy_24_neon_fast(unsigned int *src, unsigned int *dst,
 			int samples);
 
 hdmi_audio_header_t iec_header;
+EXPORT_SYMBOL(iec_header);
 
 /*
  * Note that the period size for DMA != period size for ALSA because the
-- 
1.7.9.5

